build/cli/src/arg.rs

```diff
@@ -1,3 +1,6 @@
+// === Non-Standard Linter Configuration ===
+#![allow(unused_qualifications)]
```

Comment:
To remove this attribute, we must update `clap` to the most recent version. I failed to do that, it's too dangerous to mix that with other changes. See https://github.com/clap-rs/clap/issues/4951

Reply:
Added.